pkg/iptables/iptables.go

```diff
@@ -190,3 +192,26 @@ func UpdateChainRules(ipt Interface, table, chain string, rules [][]string) erro
 
 	return nil
 }
+
+func AwaitForChain(ipt Interface, table, chain string) error {
```

Comment:
Nit: either `WaitForChain` or  `AwaitChain`.

```suggestion
func AwaitChain(ipt Interface, table, chain string) error {
```